test/verify/check-apps

```diff
@@ -318,6 +322,19 @@ This <is <not XML.
         b.wait_not_present(".pf-v5-c-empty-state")
         b.wait_visible(".app-list .pf-v5-c-data-list__item-row:contains('Summary 2')")
 
+    def testNoAppStream(self):
+        b = self.browser
```

Comment:
This shouldn't be a new test case, please just modify testBasic (as above) to check for the presence/absence of the alert.

Reply:
yep, I was wondering, I have reworked it